### ISSUE TITLE
Release 10.2.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4651]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#4651|
|Type|task|
|Created|2010-09-11 21:34:36Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-10.2.0-4651|

It's back to school time.

<details><summary>Searchable metadata</summary>

```
trac-id__4651 4651
type__task task
reporter__therve therve
priority__high high
milestone__Twisted_10_2 Twisted 10.2
branch__branches_releases_release_10_2_0_4651 branches/releases/release-10.2.0-4651
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1284240876000000 1284240876000000
changetime__1291085490000000 1291085490000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve removed owner|
|-|-|

Please review!

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @therve|
|-|-|

OK, I guess we should probably create a new branch now...

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph set owner to @glyph<br>@glyph set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_5"></a>|@glyph set owner to @exarkun<br>@glyph set status to `new`|
|-|-|

Just about ready to go...

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @glyph|
|-|-|

Looks good!  Please merge!

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph set status to `closed`|
|-|-|

(In [30363]) Release Twisted 10.2.0.

Author: glyph

Reviewer: exarkun

Fixes: [#4651](https://github.com/twisted/twisted/issues/4651)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
